bundles/binding/org.openhab.binding.plex/src/main/java/org/openhab/binding/plex/internal/PlexPlayerState.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.plex.internal;
+
+/**
+* The different states for the Plex clients  
+* 
+* @author Jeroen Idserda
+* @since 1.7.0
+*/
+public enum PlexPlayerState {
```

Comment:
btw. in ESH a new PlayerItem has been introduced. Probably it is worth porting to OH1?